tests/pactHelper.js

```diff
@@ -79,7 +79,7 @@ function setGeneralInteractions (provider) {
       }),
       headers: {
         'Access-Control-Request-Method': Pact.Matchers.regex({
-          matcher: 'GET|POST|PUT|DELETE|MKCOL|PROPFIND',
+          matcher: 'GET|POST|PUT|DELETE|MKCOL|PROPFIND|MOVE|COPY',
```

Comment:
IMO we don't need `MOVE|COPY` for now. We can add them later when we add interactions for them.

Reply:
interactions are added for move and copy request too.,